server/mock/tx.go

```diff
@@ -13,7 +13,7 @@ import (
 )
 
 // An sdk.Tx which is its own sdk.Msg.
-type kvstoreTx struct {
+type KVStoreTx struct {
```

Comment:
Why make this public?

Reply:
linter recommends it because sometimes users may need to access the type

Reply:
given that all fields are private, I don't understand why making this public. can we ignore?